<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ai_Ctl_Widget</class>
 <widget class="QWidget" name="Ai_Ctl_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intrusion_detection_and_alarm_system</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Alarm_Show</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>open service</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_btn">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>close server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="flages">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/images/off.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="listView">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Taskbar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="opentask">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>open task</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closetask">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>close task</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Logs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
